data/config.go

```diff
@@ -0,0 +1,168 @@
+package data
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kelseyhightower/envconfig"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const prefix = "TYK_K8SBOOTSTRAP"
+
+type Config struct {
+	// InsecureSkipVerify enables InsecureSkipVerify options in http request sent to Tyk - might be useful
+	// for Tyk Dashboard with self-signed certs.
+	InsecureSkipVerify bool
+	// BootstrapDashboard controls bootstrapping Tyk Dashboard or not.
+	BootstrapDashboard bool
+	// BootstrapPortal controls bootstrapping Tyk Classic Portal or not.
+	BootstrapPortal bool
+
+	// OperatorKubernetesSecretName corresponds to the Kubernetes secret name that will be created for Tyk Operator.
+	// Set it to an empty to string to disable bootstrapping Kubernetes secret for Tyk Operator.
+	// By default, tyk-operator-conf
```

Comment:
If I have understood correctly, these default values set in the helm chart.
If so, we should remove these default value comment.

Reply:
updated 👍